via/static/scripts/video_player/hooks/use-side-by-side-layout.ts

```diff
@@ -39,12 +39,16 @@ export const TOOLBAR_WIDTH = 22;
  *
  * @param [container] test seam
  */
-export function useSideBySideLayout(container = document.body) {
+export function useSideBySideLayout(container = document.body): boolean {
   const [sidebarOpen, setSidebarOpen] = useState(false);
   const [sufficientSpace, setSufficientSpace] = useState(
     container.clientWidth >= SIDEBYSIDE_THRESHOLD
   );
   const [sidebarWidth, setSidebarWidth] = useState(0);
+  const sideBySideActive = useMemo(
+    () => sufficientSpace && sidebarOpen,
+    [sufficientSpace, sidebarOpen]
+  );
```

Comment:
Since this function is really simple and just returns a boolean, so it would be fine not to memo it here.